packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardInputWarning.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import { Icon } from '../../../../index';
+import { excludeKeys } from '../../../../common/helpers';
 
 const LoginCardInputWarning = ({ children, className, ...props }) =>
   children && (
-    <div style={{ marginTop: '5px' }} className={className} {...props}>
+    <div {...excludeKeys(props, ['className'])} className={classNames('login-pf-input-warning', className)}>
```

Comment:
i can see that perspective, indeed past work can be a reference point, kudos for taking the time to research. However, if there is a more efficient way based on updated ECMA syntax, that is used in other components within this repo, past work shouldn't be the limiting factor 

`excludeKeys` really should be something we're trying to clean up, not expand. In this case it represents an oversight in how destructuring works. 

Reasons for this request are:

- an additional dependency is being created for something already baked into ecma
- a round-about-way of destructuring is still taking place

Reply:
I'll wait for @jeff-phillips-18 or @dtaylor113 opinions on this,
As I was requested to use this pattern in a very recent PR

Reply:
Hi @Laviro, I really appreciate you working though these issues with us.  Unfortunately,  @jeff-phillips-18 is out on PTO till next Tuesday I believe.  I agree with @cdcabrera that 

> `excludeKeys` really should be something we're trying to clean up, not expand. In this case it represents an oversight in how destructuring works.

In the OpenShift repos., we are always striving to use the latest ECMA syntax and promote the best and latest best practices so that all developers can adapt rather than just cut & paste what is already there (something I tend to do :-) -thanks

Reply:
thanks @dtaylor113,
I have no problem changing it, I just think we need to discuss consistency in PF,
attaching my last PR where we used `excludeKeys`: https://github.com/patternfly/patternfly-react/pull/942#discussion_r235141891

Thanks !

Reply:
we agree on the consistency @Laviro be nice to understand the full context behind the request on #942 but have to wait on that one, I'll follow up for sure with @jeff-phillips-18